app-header/app-header.html

```diff
@@ -212,24 +212,23 @@ <h1 title>App name</h1>
 
 @group App Elements
 @element app-header
-@demo app-header/demo/blend-background-1.html Blend Background Image
-@demo app-header/demo/blend-background-2.html Blend 2 Background Images
-@demo app-header/demo/blend-background-3.html Blend Background Colors
-@demo app-header/demo/contacts.html Contacts Demo
-@demo app-header/demo/give.html Resize Snapped Title Demo
+@demo app-header/demo/blend-background-1.html Blend background image
+@demo app-header/demo/blend-background-2.html Blend 2 Background images
+@demo app-header/demo/blend-background-3.html Blend background colors
+@demo app-header/demo/contacts.html Contacts demo
+@demo app-header/demo/give.html Resize snapped title demo
 @demo app-header/demo/music.html Reveals Demo
-@demo app-header/demo/no-effects.html Condenses and Reveals Demo
-@demo app-header/demo/notes.html Fixed with Dynamic Shadow Demo
+@demo app-header/demo/no-effects.html Condenses and reveals demo
+@demo app-header/demo/notes.html Fixed with Dynamic shadow demo
+@demo app-header/demo/custom-primary-element.html Custom primary element demo
```

Comment:
If we want to update the demo title to only uppercase the first word, we should do that consistently to all the demos, e.g. in [app-box.html](https://github.com/PolymerElements/app-layout/blob/269f87c0b2ad09f2e3c8a45fc4e08ff5afd6d5ae/app-box/app-box.html#L61), [app-drawer.html](https://github.com/PolymerElements/app-layout/blob/269f87c0b2ad09f2e3c8a45fc4e08ff5afd6d5ae/app-drawer/app-drawer.html#L50)...